test/microbench/fam-api-mb/fam_microbenchmark_allocator.cpp

```diff
@@ -319,8 +319,13 @@ TEST(FamChangePermissions, DataItemChangePermission) {
     Fam_Descriptor *itemLocal;
     const char *testRegionLocal = get_uniq_str("testLocal", my_fam);
     const char *firstItemLocal = get_uniq_str("firstLocal", my_fam);
-    EXPECT_NO_THROW(descLocal = my_fam->fam_create_region(
+    try {
+         EXPECT_NO_THROW(descLocal = my_fam->fam_lookup_region(testRegionLocal));
```

Comment:
i think you will need to remove EXPECT_NO_THROW around fam_lookup_region, test case will fail - seeing an expect no throw and try catch around is not useful.